ui/components/common/CodeEditor.js

```diff
@@ -79,28 +74,20 @@ const CodeEditor = class extends Component {
     }
 
     render() {
-        const { editorValue, lineHeight } = this.state;
+        const { editorValue } = this.state;
         return (
-            <div className={style["editor-container"]}>
+            <div className={style.editorContainer}>
```

Comment:
@josemigallas  is this way of referencing css preferred to this? `{style["editor-container"]}`

Reply:
The only different is how to access the property (period or brackets).

Reply:
👍

Reply:
@cfoskin Javascript names can't have dashes, so when your class has a dash in the name you need to use a string instead of the dot notation.